docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -308,6 +308,13 @@ Then install Ansible [1]_::
 
     $ python -m pip install --user ansible
 
+
+.. note::
+
+    When installing Python packages with ``pip install --user``, binaries will be placed into ``$HOME/.local/bin``. If this is your first time installing packages in this fashion, you will need to add this directory to your ``$PATH``. For example, you could add the following to your ``$HOME/.bashrc`` file::
```

Comment:
This is not always correct. The directory where `pip` installs with `--user` is based on `site.USER_BASE`. For example, on macOS this is `~/Library/Python/2.7/bin/`. Maybe it is better to indicate how to get this path dynamically rather than indicating a value here that may not always be correct.

`python -c 'import site; print(site.USER_BASE)'`

Reply:
Something else is going on here.  Even when you find the path dynamically, pip is not actually creating anything there that can be used to invoke ansible

Reply:
@sfermigier it does but there's a pip bug that is related to the upgrade scenario that results in some of the package contents being removed right after it's installed because of the package name changes.